### PR TITLE
Split namespace docs user vs admin.

### PR DESCRIPTION
**Pull request by erictune**
Move namespace.md and examples dir from docs/user-guide
to docs/admin.

Assumption is that creating and deleting namespaces is an "admin"
task.

Add a mostly new user-guide to namespaces that gives more advice
on when to use namespaces, and how to work within them,
but not how to create/delete them.  It is more succinct than
before.

### CODE REVIEW

File: docs/user-guide/namespaces.md [hunk 1]
```diff
@@ -196,36 +111,12 @@ is local to a namespace.  This is useful for using the same configuration across
...
-
-## Design
+## Not All Objects are in a Namespace
 
-Details of the design of namespaces in Kubernetes, including a [detailed example](../design/namespaces.md#example-openshift-origin-managing-a-kubernetes-namespace)
-can be found in the [namespaces design doc](../design/namespaces.md)
+Most kubernetes resources (e.g. pods, services, replication controllers, and others) are
+in a some namespace.  However namespace resources are not themselves in a namespace.
+And, low-level resources, such as [nodes](../../docs/admin/node.md) and
+persistentVolumes, are not in any namespace.
```
Comment (lavalamp) [inline docs/user-guide/namespaces.md]: Add something like: Events are an exception: they may or may not have a namespace, depending on the object the event is about.

**Comment by lavalamp** [comment]
LGTM, comment is optional

**Comment by erictune** [comment]
addressed comment.

**Comment by lavalamp** [comment]
hack/run-gendocs.sh

**Comment by erictune** [comment]
Fixed by pushing the file I forgot to push in last PR.

**Comment by lavalamp** [comment]
LGTM

**Comment by krousey** [comment]
@erictune I think this is or is going to break the build. You moved example files without updating the examples test. Could you please send a follow up PR asap?

**Pull request merged**